src/mobu/constants.py

```diff
@@ -5,13 +5,17 @@
 from datetime import timedelta
 
 __all__ = [
+    "NOTEBOOK_ALWAYS_EXCLUDE_DIRS",
     "NOTEBOOK_REPO_URL",
     "NOTEBOOK_REPO_BRANCH",
     "TOKEN_LIFETIME",
     "USERNAME_REGEX",
     "WEBSOCKET_OPEN_TIMEOUT",
 ]
 
+NOTEBOOK_ALWAYS_EXCLUDE_DIRS = {"mobu_exclude"}
+"""NotebookRunner always excludes notebooks in these dirs."""
+
```

Comment:
This feels a little magical and hard-coded to me. I think I would lean towards not excluding any directories by default and making people add an excluded directory if they want one. But maybe I'm missing a use case for always having a default excluded directory?

Reply:
Agreed it seems magical.  There are a couple of issues:
1. One of the things we want is for folks to be able to manage these excluded notebooks without having to make phalanx PRs.  IIUC, all of the flock config is defined in phalanx, so folks would have to make phalanx PRs as they add and remove excluded dirs (or make one phalanx MR to get their own single exclude dir like in this magic default)
2. A magic default exclude dir gets us out of having to tightly coordinate this mobu release with notebook repo changes.  With this, we can [modify the notebook repos](https://github.com/lsst-sqre/system-test/pull/84) any time before the mobu release to any environment without having to coordinate adding the `exclude_dirs` flock option at the same time as we release to each environment.
    * To address this, we could have this behavior be opt-in by adding a `run_in_subdirs` option that defaults to False.  Then we can deploy the mobu change without affecting current behavior.  We could then change the default at some point in the future when all of the flocks have their excluded dirs specified appropriately.

One bigger change we could make to address both of these issues is to have this behavior specified in a config file or something in the notebook repos themselves.  Then:
* We could first add that config file to the repos before deploying the mobu changes and current behavior would be kept everywhere
* No phalanx changes would be needed to manage these exclude dirs

Reply:
I ditched the magic dir.  Seems not worth the magic and confusion.  I add the subdirs in the currently existing flocks in the [phalanx PR](https://github.com/lsst-sqre/phalanx/pull/3346).

Reply:
And I think I misunderstood the way the release process will work.  The mobu version and the flock config will get synced together for each environment; there won't be a state where mobu is updated but the flock config isn't.